packages/@react-spectrum/list/test/ListViewDnd.test.js

```diff
@@ -428,10 +429,15 @@ describe('ListView', function () {
         fireEvent(grid, new DragEvent('drop', {dataTransfer, clientX: 1, clientY: 110}));
         act(() => jest.runAllTimers());
         await act(async () => Promise.resolve());
+
         expect(onDrop).toHaveBeenCalledTimes(1);
 
         fireEvent(cell, new DragEvent('dragend', {dataTransfer, clientX: 1, clientY: 110}));
-        expect(onDragEnd).toHaveBeenCalledTimes(1);
+        if (isReact19) {
+          expect(onDragEnd).toHaveBeenCalledTimes(2);
```

Comment:
I guess this is the react bug we filed? Should we have a todo somewhere to fix it later?